### PR TITLE
Abstract for FortranCon 2020 about stdlib

### PR DESCRIPTION
**Pull request by jvdp1**
Edited after comments:

**Authors**:
[Jeremie Vandenplas](https://github.com/jvdp1), jeremie.vandenplas@wur.nl, Wageningen University & Research
[Bálint Aradi](https://github.com/aradi), aradi@uni-bremen.de, University of Bremen
[Izaak Beekman](https://github.com/zbeekman), contact@izaakbeekman.com, ParaTools Inc.
[Ondrej Certik](https://github.com/certik), ondrej@certik.us, Los Alamos National Laboratory
[Milan Curcic](https://github.com/milancuric), caomaco@gmail.com, University of Miami
[Pierre de Buyl](https://github.com/pdebuyl), pdebuyl@pdebuyl.be, KU Leuven
[Juan Fiol](https://github.com/fiolj),,
[Michael Hirsch](https://github.com/scivision), info@scivision.dev, SciVision, Inc.
[Ivan Pribec](https://github.com/ivan-pi), ivan.pribec@tum.de, Technical University of Munich
[Nathaniel Shaffer](https://github.com/nshaffer), nrshaffer@protonmail.com, Los Alamos National Laboratory

**To be acknowledged in the presentation (following comments):**
[Neil Carlson](https://github.com/nncarlson)
[Pedro Costa](https://github.com/p-costa)
[???](https://github.com/JHenneberg)
[Yuichiro Sakamoto](https://github.com/sakamoti)
[Ashwin Vishnu](https://github.com/ashwinvis)

**Track**: Junior presentations (15 + 5 minutes discussion)

**Presenter**: [Jeremie Vandenplas](https://github.com/jvdp1)

**Title**: Fortran Standard Library

**Alternative title**: Fortran Standard Library - a community project

Deadline: June 1

### CODE REVIEW

File: FortranCon2020-stdlib/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+**Presenter**: To be confirmed
+
+**Title**: Fortran Standard Library
+
+**Alternative title**: Fortran Standard Library - a community project
+
+**Abstract**:
+The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
+Therefore, the goal of this project is to provide a community driven and agreed upon de facto "standard" library for Modern Fortran, called Fortran Standard Library (stdlib; https://github.com/fortran-lang/stdlib).
```
Comment (certik) [inline FortranCon2020-stdlib/README.md]: I am not a native speaker, shouldn't there be *the*:
```suggestion
Therefore, the goal of this project is to provide a community driven and agreed upon de facto "standard" library for Modern Fortran, called the Fortran Standard Library (stdlib; https://github.com/fortran-lang/stdlib).
```

File: FortranCon2020-stdlib/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+This library aims to provide to the community a set of procedures for science, engineering, and mathematics. Among others, it will include procedures for manipulating strings and files, for assertions and unit testing, but also, e.g., for searching and sorting algorithms, linear algebra, sparse matrices, or statistics.
+Started a few months ago, no less than 15 people already contributed to the development of the Fortran Standard Library and its documentation.
+Currently the library includes procedures for catching and handling errors, handling optional arguments, facilitating I/O operations, linear algebra, numerical integration, and descriptive statistics.
+Many other programmers are also involved in active discussions about the development of the Fortran Standard Library though GitHub issues.
```
Comment (certik) [inline FortranCon2020-stdlib/README.md]:
```suggestion
Many other programmers are also involved in active discussions about the development of the Fortran Standard Library through GitHub issues.
```

File: FortranCon2020-stdlib/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+**Presenter**: To be confirmed
+
+**Title**: Fortran Standard Library
+
+**Alternative title**: Fortran Standard Library - a community project
+
+**Abstract**:
+The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
+Therefore, the goal of this project is to provide a community driven and agreed upon de facto "standard" library for Modern Fortran, called Fortran Standard Library (stdlib; https://github.com/fortran-lang/stdlib).
+This library aims to provide to the community a set of procedures for science, engineering, and mathematics. Among others, it will include procedures for manipulating strings and files, for assertions and unit testing, but also, e.g., for searching and sorting algorithms, linear algebra, sparse matrices, or statistics.
```
Comment (certik) [inline FortranCon2020-stdlib/README.md]: I would maybe add here:
```
The overall scope of stdlib is roughly as that of SciPy or the default built-in Matlab scientific environment.
```

**Review by certik — APPROVED** [review]
I think this looks great. Thanks for writing it! I left some minor comments.

File: FortranCon2020-stdlib/README.md (hunk 3 above)
Comment (milancurcic) [reply FortranCon2020-stdlib/README.md]: :+1: but I think it needs:

> The overall scope of stdlib is roughly as **that of** SciPy or...

**Review by milancurcic — APPROVED** [review]
Thank you, @jvdp1! Looks good to me, pending @certik's suggestions which I agree with.

File: FortranCon2020-stdlib/README.md (hunk 3 above)
Comment (certik) [reply FortranCon2020-stdlib/README.md]: Ok, I added that.

Comment (pdebuyl) [reply FortranCon2020-stdlib/README.md]: suggestion:
> The overall scope of stdlib is similar to the one of SciPy or to the default built-in Matlab scientific environment.

**Comment by pdebuyl** [comment]
Hi @jvdp1 thank you for the initiative!

**Comment by jvdp1** [comment]
Thank you @certik @milancurcic @pdebuyl for the comments. They are now integrated.

**Comment by milancurcic** [comment]
Thanks Jeremie. You can merge yourself when you're ready.

**Comment by jvdp1** [comment]
> Thanks Jeremie. You can merge yourself when you're ready.

I will let it here a few more days, such that other contributors have a chance to read it.

Regarding co-authors, I don't have full names, affiliations or/and email addresses of some of those listed. what is the plan for that?

**Comment by milancurcic** [comment]
Sounds good. Ultimately, it will be up to you, but here's what I think makes most sense. You invited all contributors to be co-authors and that's what matters. Those who want to be co-authors and contribute to the talk, they'll write here and provide the needed information. Everybody else, you can acknowledge in the talk. This is how I plan to do it for the community talk.

**Comment by ashwinvis** [comment]
Thanks to @fortran-lang developers for being inclusive and welcoming. I appreciate the gesture, but I do not consider my *small patch* worthy to be counted amongst co-authors. Feel free to exclude my name.

 +1 on @milancurcic 's point

**Comment by p-costa** [comment]
I also think that my very, very minor contributions are not enough to deserve co-authorship (or even acknowledgment), so please feel free to exclude my name.

**Comment by sakamoti** [comment]
Thank you for inviting me to be one of a co-authors. I only tested the library and fixed the specific issue, so I don't think it's enough to participate as a co-author now. 
I'm very interested in this project, so if it's as an acknowledgement, I'm glad to be included my name. 
(I've added my name to my profile on github.)

File: FortranCon2020-stdlib/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# FortranCon2020-stdlib
+
+**Authors**:
+[Balint Arady](https://github.com/aradi)
```
Comment (aradi) [inline FortranCon2020-stdlib/README.md]: [Bálint Aradi](https://github.com/aradi)

**Review by aradi — APPROVED** [review]
Thanks a lot to include me on the author list! I think, it is a very good idea to present the stdlib on the FortranCon 2020!

**Comment by JHenneberg** [comment]
> I also think that my very, very minor contributions are not enough to deserve co-authorship (or even acknowledgment), so please feel free to exclude my name.

Same for me.

File: FortranCon2020-stdlib/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+**Alternative title**: Fortran Standard Library - a community project
+
+**Abstract**:
+The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
+Therefore, the goal of this project is to provide a community driven and agreed upon de facto "standard" library for Modern Fortran, called the Fortran Standard Library (stdlib; https://github.com/fortran-lang/stdlib).
+This library aims to provide to the community a set of procedures for science, engineering, and mathematics.
+The overall scope of the Fortran Standard Library is therefore similar to the one of SciPy or to the default built-in Matlab scientific environment.
+Started a few months ago, no less than 15 people already contributed to the development of the Fortran Standard Library and its documentation.
+Currently the library includes procedures for catching and handling errors, handling optional arguments, facilitating I/O operations, linear algebra, numerical integration, and descriptive statistics.
```
Comment (nshaffer) [inline FortranCon2020-stdlib/README.md]:
```suggestion
Currently the library includes procedures for catching and handling errors, handling optional arguments, facilitating I/O operations, linear algebra, numerical integration, and descriptive statistics.
Started a few months ago, no less than 15 people already contributed to the development of the Fortran Standard Library and its documentation.
```
Seems like these two sentences should be swapped.

File: FortranCon2020-stdlib/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+[Ondrej Certik](https://github.com/certik)
+[Pedro Costa](https://github.com/p-costa)
+[Milan Curcic](https://github.com/milancuric)
+[Pierre de Buyl](https://github.com/pdebuyl)
+[???](https://github.com/fiolj)
+[???](https://github.com/JHenneberg)
+[Michael Hirsch](https://github.com/scivision)
+[Ivan Pribec](https://github.com/ivan-pi)
+[???](https://github.com/sakamoti)
+[???](https://github.com/nshaffer)
```
Comment (nshaffer) [inline FortranCon2020-stdlib/README.md]:
```suggestion
[Nathaniel Shaffer](https://github.com/nshaffer)
```

File: FortranCon2020-stdlib/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+**Track**: Junior presentations (15 + 5 minutes discussion)
+
+**Presenter**: To be confirmed
+
+**Title**: Fortran Standard Library
+
+**Alternative title**: Fortran Standard Library - a community project
+
+**Abstract**:
+The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
```
Comment (nshaffer) [inline FortranCon2020-stdlib/README.md:31]: Maybe add a sentence like:
```suggestion
The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
The language can be extended with new intrinsic procedure and modules, but these must be formally standardized and then implemented by compiler vendors before becoming available to users.
```
Maybe add a sentence for context? Not sure about word limits.

File: FortranCon2020-stdlib/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+Therefore, the goal of this project is to provide a community driven and agreed upon de facto "standard" library for Modern Fortran, called the Fortran Standard Library (stdlib; https://github.com/fortran-lang/stdlib).
+This library aims to provide to the community a set of procedures for science, engineering, and mathematics.
+The overall scope of the Fortran Standard Library is therefore similar to the one of SciPy or to the default built-in Matlab scientific environment.
+Started a few months ago, no less than 15 people already contributed to the development of the Fortran Standard Library and its documentation.
+Currently the library includes procedures for catching and handling errors, handling optional arguments, facilitating I/O operations, linear algebra, numerical integration, and descriptive statistics.
+Many other programmers are also involved in active discussions about the development of the Fortran Standard Library through GitHub issues.
+The development of this library is part of the Fortran-lang project and aims to collaborate with the Fortran Standards Committee.
```
Comment (nshaffer) [inline FortranCon2020-stdlib/README.md:39]: Do we want to be more specific? My understanding is that stdlib is to be the reference implementation for compiler vendors if is eventually adopted by the standards committee.

**Review by nshaffer — COMMENTED** [review]
Great initiative on getting an abstract up. I think FortranCon is a perfect place to publicize stdlib. Just filled in my name and had some minor suggestions. Take them or leave them.

**Comment by nncarlson** [comment]
> I also think that my very, very minor contributions are not enough to deserve co-authorship (or even acknowledgment), so please feel free to exclude my name.

And same for me as well. But your gesture to be inclusive is appreciated. Hopefully sometime in the not so distant future I'll have the time to contribute in a more significant way to this important effort.

File: FortranCon2020-stdlib/README.md (hunk 4 above)
Comment (jvdp1) [inline FortranCon2020-stdlib/README.md]:
```suggestion
[Bálint Aradi](https://github.com/aradi)
```

File: FortranCon2020-stdlib/README.md (hunk 8 above)
Comment (jvdp1) [reply FortranCon2020-stdlib/README.md:39]: I keep your suggestion for the presentation. I can't integrate this in the abstract due to word limits

**Comment by jvdp1** [comment]
Thank you all for your comments. I will submit the abstract tomorrow afternoon.

**Comment by jvdp1** [comment]
I submitted the manuscript. Thank you all for your comments and input.

**Pull request merged**